.circleci/unittest/windows/scripts/setup_env.sh

```diff
@@ -45,3 +45,7 @@ printf "* Downloading SpaCy English models\n"
 python -m spacy download en_core_web_sm
 printf "* Downloading SpaCy German models\n"
 python -m spacy download de_core_news_sm
+
+# 6. Post install script for pywin32
+curl --output pywin32_postinstall.py https://raw.githubusercontent.com/mhammond/pywin32/main/pywin32_postinstall.p
```

Comment:
It seems there is a missing `y` at the end.

Reply:
oops thanks for the catch!